### ISSUE TITLE
Change default step size of position distance to 0.5 meters

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#121]
As discussed on Monday, the default step size of 0.2 meters is too small for keyboard users to make productive changes.

Please change default step size to  0.5 meters.

If not too difficult, please provide a query parameter that allows me to change the step size (e.g. to 0.4, 0.6, or 0.8).

**Comment by terracoda** [comment]
Oops, wrong button. I didn't mean to close the issue.

**Comment by terracoda** [comment]
@mbarlow12, thanks for adding the query parameters.

It is safe to change the actual default step size to 0.5 meters for the position sliders. I ran this by Ariel and Kathy yesterday and they were fine with 0.5 meters as the default step size.

Please keep the query parameter for now in case we need to play with other values.

**Comment by zepumph** [comment]
From meeting with @terracoda tonight:

0.5 meters in GFL
0.5 km in GFL:B

**Comment by terracoda** [comment]
@zepumph, the default step size using the keyboard is working fine in GFL REGULAR, but there are some serious bugs in GFL BASICS.
- I can't move either sphere with the keyboard using the arrow keys

In addition (I think this might be unrelated) when I press an arrow key to move a sphere closer and the sphere does not move, I get  an incorrect alert that the spheres are extremely close, but they are neither extremely close, nor has the sphere moved. 

I'll have to make a separate issue because I need to re-think that state alert all together. **However, I am curious as to why the incorrect described distance is being read out.** Maybe it is just a static string which is triggered when a sphere does not move ans is assuming they spheres are next to each other.

This "state alert" was implemented to compensate for the fact that `aria-valuetext` is not read out if position does not change (e.g., when spheres are next to each other or at an edge). I'll be re-thinking this state alert once as I review the position alerts, distance, and the position edge cases.

<img width="1104" alt="bug-cant-move-sphere-extremly-close-alert" src="https://user-images.githubusercontent.com/1902116/51754447-9d821800-20bc-11e9-9f70-63e1e3611b08.png">

@zepumph, do you think it is safe to close this issue in REGULAR and open a new issue in BASICS?

**Comment by zepumph** [comment]
I'm sorry about that! I thought we could use the same value but we couldn't. It should be fixed now. Please verify it is working as you would expect, and that there is nothing left to do for this issue.

**Comment by terracoda** [comment]
@zepumph, perfect! You fixed it. Closing issue.

You are right, the scales are a bit different in the 2 sims. I think this because mass in BASICS only has 10 values (1-10 billion kilograms) whereas in REGULAR mass has up to 1000 values (1-1000 kg). I haven't sorted out the differences in my head yet.
